src/main/webapp/js/form.js

```diff
@@ -1,6 +1,31 @@
+$("#randomize-form").submit(function(event) {
+    const errorEl = document.getElementById("error");
+    errorEl.classList.add("hidden");
+
+    event.preventDefault();
+    let url = $(this).attr("action");
+    let lat = localStorage.getItem("lat");
+    let lng = localStorage.getItem("lng");
+    let userID = 0;
+    if (localStorage.getItem("loggedIn")) {
+        userID = localStorage.getItem("user");
+    }
+    let queryStr = $(this).serialize() + `&lat=${lat}&lng=${lng}&user=${userID}`;
+    query(queryStr);
+});
+
+// Form underline element
+$("input, textarea").blur(function() {
+    if ($(this).val() != "") {
+        $(this).addClass("active");
+    } else {
+        $(this).removeClass("active");
+    }
+});
+
 //This file handles the formatting for the form on RRP Homepage
 //This is done with a third-party formatting and styling package
-(function ($) {
+(function($) {
     'use strict';
     /*==================================================================
         [ Daterangepicker ]*/

```

Comment:
where exactly is daterangepicker used in RRP?

Reply:
Kept in for a future search scheduling feature but is still unused. I'll remove it for now.